wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/document/EventLogScreen.kt

```diff
@@ -21,75 +23,203 @@ import androidx.compose.ui.res.stringResource
+    val eventInfos = remember { documentModel.getEventInfos(documentId) }
+    val eventInfo = eventInfos.find { it.timestamp == timestamp }
+
+    ScreenWithAppBarAndBackButton(
+        title = "",
+        onBackButtonClick = { navController.popBackStack() }
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .weight(1f)
+        ) {
+            if (eventInfo == null) {
+                Text(text = "Event not found")
+            } else {
+                Text(
+                    text = documentModel.getDocumentInfo(documentId)?.name ?: "Unknown",
+                    modifier = Modifier.fillMaxWidth(), // Fill the width
+                    textAlign = TextAlign.Center // Center the text horizontally
+                )
+                Spacer(modifier = Modifier.height(8.dp))
+
+                val requesterInfo = eventInfo.requesterInfo ?: EventLogger.RequesterInfo(
+                    type = EventLogger.Requester.Anonymous(),
+                    shareType = EventLogger.ShareType.UNKNOWN
+                )
+
+                val shareTypeText = when (requesterInfo.shareType) {
+                    EventLogger.ShareType.SHARED_IN_PERSON -> stringResource(id = R.string.document_info_screen_menu_activities_log_screen_shared_in_person)
+                    EventLogger.ShareType.SHARED_WITH_APPLICATION -> stringResource(id = R.string.document_info_screen_menu_activities_log_screen_shared_with_application)
+                    EventLogger.ShareType.SHARED_WITH_WEBSITE -> stringResource(id = R.string.document_info_screen_menu_activities_log_screen_shared_with_website)
+                    EventLogger.ShareType.UNKNOWN -> stringResource(id = R.string.document_info_screen_menu_activities_log_screen_unknown_share)
+                }
+
+                KeyValuePairText(
+                    stringResource(id = R.string.document_info_screen_menu_activities_log_screen_share_type),
+                    shareTypeText
+                )
+                Spacer(modifier = Modifier.height(8.dp))
+                KeyValuePairText(
+                    stringResource(id = R.string.document_info_screen_menu_activities_log_screen_requester),
+                    requesterInfo.type.toString()
+                )
+                Spacer(modifier = Modifier.height(8.dp))
+
+                val fieldsList = eventInfo.requestedFields.replace("_", " ")
+                    .split(",") // Replace underscores with spaces
+
+                KeyValuePairText(
+                    stringResource(id = R.string.document_info_screen_menu_activities_log_screen_data_shared),
+                    ""
+                )
+                Card(
+                    modifier = Modifier
+                        .padding(16.dp)
+                        .fillMaxWidth(),
+                    shape = RoundedCornerShape(16.dp),
+                ) {
+                    Column {
```

Comment:
Add maybe 8.dp or 16.dp padding on the left? From the screenshot, the text is right up against the rounded card, maybe even getting clipped by the rounded corners.

Reply:
Done.